src/Microsoft.DocAsCode.Build.Engine/TemplateProcessors/TemplateManager.cs

```diff
@@ -43,6 +43,9 @@ public string GetTemplatesHash()
             {
                 return null;
             }
+
+            Logger.LogVerbose("Calculating template hash...");
```

Comment:
this line seems useless

Reply:
Just to be consistent with plugin hash logs. We can also remove them later.